src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/BulkPrintService.java

```diff
@@ -168,4 +233,8 @@ private BulkPrintDocument generateRespondentCoverSheet(FinremCaseDetails caseDet
 
         return documentHelper.getCaseDocumentAsBulkPrintDocument(respondentCoverSheet);
     }
+
+    public String getRecipient(String text) {
+        return StringUtils.remove(WordUtils.capitalizeFully(text, '_'), "_");
```

Comment:
Can be use StringUtils.capitalize, would recon to use Springs StringUtils so no need to import apache common